pkg/daemon/controller/ipinstance.go

```diff
@@ -37,6 +37,18 @@ import (
 	"k8s.io/klog"
 )
 
+func (c *Controller) filterIPInstance(obj interface{}) bool {
+	p, ok := obj.(*networkingv1.IPInstance)
+	if !ok {
+		return false
+	}
+
+	nodeLabel := p.Labels[constants.LabelNode]
+
+	// only ip on this node
+	return len(nodeLabel) == 0 || nodeLabel == c.config.NodeName
```

Comment:
Only IPInstances with valid node should be processed.

Reply:
I thought you mean that a node label of an ip instance could be empty xD.  What did the "should empty string be ignored here?" means in the old conversation?  Maybe you want `len(nodeLabel) != 0 && nodeLabel == c.config.NodeName`?